Update job description for Software Engineer iOS

packatino opened the pull request.

packatino committed:
Update Software Engineer iOS.md

packatino committed:
Fix typo

Sebruck commented on Software Engineer iOS.md:
```diff
@@ -29,21 +29,14 @@
 ## Apply
 
-If you happen to be a pragmatic, test-driven software craftsman or craftswoman who likes working on the edge of technology, come and help our outstanding, truly agile development teams in [Munich, DE](https://goo.gl/maps/2KKGh), in [Bottighofen, CH](https://goo.gl/maps/X7bZ3), in [Poznan, PL](https://goo.gl/maps/AiHKJ), or in [Amsterdam, NL](https://goo.gl/maps/AJHpM3yYUzL2).
+If you happen to be a pragmatic, test-driven software craftsman or craftswoman who likes working on the edge of technology, come and help our outstanding, truly agile development team at Lake Constance in [Bottighofen, CH](https://goo.gl/maps/X7bZ3).
```
Instead of craftsmen & craftswoman which excludes people who doesn’t see themselves as man or woman, you could use Software Crafter. We use the same wording on https://apprenticeship.holidaycheck.com

packatino replied:
Thanks for the hint. I pushed a new commit which tackles this issue.

packatino committed:
Update Software Engineer iOS.md

Since we want to have a diverse team at HolidayCheck, I replaced two gender-specific words with a neutral one. While it might seem like an unimportant change to many people it actually makes a huge difference for some other people.

Sebruck reviewed (approved):
LGTM 👍
